src/OrbitGl/IntrospectionWindow.cpp

```diff
@@ -13,7 +13,187 @@ using orbit_client_data::CaptureData;
 using orbit_client_protos::TimerInfo;
 using orbit_grpc_protos::CaptureStarted;
 
-IntrospectionWindow::IntrospectionWindow(OrbitApp* app) : CaptureWindow(app) {
+namespace {
+inline void HandleCaptureEvent(const orbit_api::ApiScopeStart& scope_start,
+                               orbit_capture_client::ApiEventProcessor* api_event_processor) {
+  orbit_grpc_protos::ApiScopeStart api_event;
+  scope_start.CopyToGrpcProto(&api_event);
+  api_event_processor->ProcessApiScopeStart(api_event);
+}
+
+inline void HandleCaptureEvent(const orbit_api::ApiScopeStop& scope_stop,
```

Comment:
does inline actually give us anything here? I suspect it is inlined anyways and we should probably leave it to compiler to decide. Unless this is absolutely necessary.

Reply:
Removed it.